### DISCUSSION TITLE
Open Display Settings page

### DISCUSSION THREAD

**Discussion by nijel** [Blackymas/NSPanel_HA_Blueprint#2233] [Q&A] [answered]
Wi-Fi got stuck on my panel, and I'd like to reboot it (yes, I can do it by switching off the mains, but that power cycles other things as well what I'd like to avoid). There appears to be that option in the UI:

![Settings EU](https://raw.githubusercontent.com/Blackymas/NSPanel_HA_Blueprint/main/docs/pics/eu_settings.png)

But how do I open this page on the panel? It would be useful to mention that at https://github.com/Blackymas/NSPanel_HA_Blueprint?tab=readme-ov-file#display-settings

**Comment by edwardtfn** [comment, marked as answer]
A long press on the area showing the time will bring you to that screen.
